### ISSUE TITLE
When change LOSS_GAMMA to 0.0 and LOSS_ALPHA to 0.5, retinanet loss go to NaN

### ISSUE THREAD

**Issue by pandazha** [facebookresearch/Detectron#356]

**Comment by DonaldTang1995** [comment]
The focal (softmax or sigmoid) loss easily goes to NaN from my experiments. (Can't explain why) Try to decrease the base_lr. (The performance may decrease too)

**Comment by liujing1995** [comment]
I have met the same question and I chose different pre-trained model to fix this .
